pipeline-model-definition/src/main/groovy/org/jenkinsci/plugins/pipeline/modeldefinition/Utils.groovy

```diff
@@ -173,27 +172,12 @@ class Utils {
         }
     }
 
+    /**
+     * Use {@link ASTParserUtils#isStageWithOptionalName} instead.
```

Comment:
`@deprecated Use {@link ASTParserUtils#isStageWithOptionalName} instead.` will make javadoc generation handle it a bit smarter.